opt_einsum/contract.py

```diff
@@ -375,7 +375,7 @@ def _tensordot(x, y, axes, backend='numpy'):
 # Rewrite einsum to handle different cases
 def contract(*operands, **kwargs):
     """
-    contract(subscripts, *operands, out=None, dtype=None, order='K', casting='safe', use_blas=True, optimize=True, memory_limit=None, backend='numpy')
+    contract(subscripts, *operands, out=None, dtype=None, order='K', casting='safe', use_blas=True, optimize=True, memory_limit=None, backend='numpy') 
```

Comment:
A space was ended at the end here and should be removed.

Reply:
done